### PR TITLE
Add favicon, technical interview stage, and fix typos

### PR DESCRIPTION
**Pull request by WillemGerritzen**

### CODE REVIEW

File: src/config.adoc [hunk 1]
```diff
@@ -1,3 +1,4 @@
 :imagesdir: images
 :nofooter:
 :toc-title: Overview
+:favicon: images/favicon.ico
```
Comment (Stephan202) [inline src/config.adoc]: So far the entries were sorted. Shall we retain that?

File: src/pages/hiring_process.adoc [hunk 2]
```diff
@@ -61,34 +61,58 @@ sure you're able to give us a quick explanation.
 
 image::ghhp3.png[Process_3]
 
-=== Assignment
+=== Assignment & interview
 
-* Duration: ~ 1 week (highly dependent on speed)
-* What is expected of you:
+* Duration: 1 to 2 weeks (dependent on speed)
+* Assignment:
```
Comment (Stephan202) [inline src/pages/hiring_process.adoc:67]: This line doesn't seem to add anything. Can't we just omit it?

File: src/pages/hiring_process.adoc [hunk 3]
```diff
@@ -61,34 +61,58 @@ sure you're able to give us a quick explanation.
...
-** Quality of code: The main goal of the assignment is to assess your
+*** Quality of code: The main goal of the assignment is to assess your
 technical abilities when you're performing at your best. Make sure to
 give yourself the right environment to write the best code you can, and
 don't hesitate to pull out all the stops to impress the tech team. But
 remember, at Picnic, we like efficient asnwers, so avoid
 over-engineering. 😉
-
-** Time limit: Officially, there is no deadline. Your time to submit
++
```
Comment (Stephan202) [inline src/pages/hiring_process.adoc]:
```suggestion

```
There are several more of these stray `+`-es below.

File: src/pages/hiring_process.adoc [hunk 4]
```diff
@@ -61,34 +61,58 @@ sure you're able to give us a quick explanation.
...
 Once you've submitted your assignment, two members of the tech team will
 take a look independently, and then join and discuss. They will then
 share their feedback with your recruiter, and together, they will decide
-whether to invite you to the last stage of the process: an on-site
-visit.
+whether to invite you to the next stage: the technical interview.
+
+* Interview
+
+** What is expected of you:
```
Comment (Stephan202) [inline src/pages/hiring_process.adoc:89]: Throughout: do the trailing colons really add anything?

**Review by Stephan202 — COMMENTED** [review]
Quick skim :)

**Comment by WillemGerritzen** [comment]
`+`

**Review by Mordavolt — APPROVED** [review]
LGTM :)

**Pull request merged**